File: Selenium/Activities/Activity19.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class Activity19 {
    public static void main(String[] args) {
        WebDriverManager.chromedriver().setup();
        WebDriver driver = new ChromeDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("Home page title: " + driver.getTitle());
        driver.findElement(By.xpath("//button[@id='confirm']")).click();
        String alertText=driver.switchTo().alert().getText();
        System.out.println("Text of alert : "+alertText);
        driver.switchTo().alert().accept();
        driver.findElement(By.xpath("//button[@id='confirm']")).click();
        driver.switchTo().alert().dismiss();
        driver.close();
    }

}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as chromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver import ActionChains
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.common.alert import Alert
service=chromeService(ChromeDriverManager().install())
with webdriver.Chrome(service=service) as driver:
    actions = webdriver.ActionChains(driver)
    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("Page title is: ", driver.title)

    driver.find_element(By.XPATH,"//button[@id='confirm']").click()
    alertText=driver.switch_to.alert.text
    print("Text of alert : "+alertText)
    driver.switch_to.alert.accept()
    driver.find_element(By.XPATH,"//button[@id='confirm']").click();
    driver.switch_to.alert.dismiss()
    driver.close();
